packages/docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/sdk/Swift/Logto/Enums/LogtoCore.md

```diff
@@ -0,0 +1,135 @@
+---
+sidebar_position: 2
+---
+
+**枚举**
+
+# `LogtoCore`
+
+```swift
+public enum LogtoCore
+```
+
+## 结构体
+###   [CodeTokenResponse](../Structs/LogtoCore.CodeTokenResponse.md)
+###   [OidcConfigResponse](../Structs/LogtoCore.OidcConfigResponse.md)
+###   [RefreshTokenTokenResponse](../Structs/LogtoCore.RefreshTokenTokenResponse.md)
+###   [UserInfoResponse](../Structs/LogtoCore.UserInfoResponse.md)
+
+## 属性
+### `postHeaders`
+
+```swift
+static let postHeaders: [String: String] = [
+    "Content-Type": "application/x-www-form-urlencoded",
+]
+```
+
+## 方法
+### `fetchOidcConfig(useSession:uri:)`
+
+```swift
+static func fetchOidcConfig(
+    useSession session: NetworkSession = URLSession.shared,
+    uri: URL
+) async throws -> OidcConfigResponse
+```
+
+### `fetchToken(useSession:byAuthorizationCode:codeVerifier:tokenEndpoint:clientId:redirectUri:)`
+
+```swift
+static func fetchToken(
+    useSession session: NetworkSession = URLSession.shared,
+    byAuthorizationCode code: String,
+    codeVerifier: String,
+    tokenEndpoint: String,
+    clientId: String,
+    redirectUri: String
+) async throws -> CodeTokenResponse
+```
+
+Fetch token by `authorization_code`.
+The returned `access_token` is only for user info enpoint.
```

Comment:
typo `endpoint`

Reply:
Also should it be `user info` or `userinfo`?

Reply:
actually UserInfo Endpoint. updated